app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/AllergensAlertFragment.java

```diff
@@ -90,6 +90,7 @@ public void onViewCreated(@NonNull final View view, @Nullable Bundle savedInstan
         productRepository = ProductRepository.getInstance();
         mDataObserver = new DataObserver();
         bottomNavigationView  = view.findViewById((R.id.bottom_navigation));
+        bottomNavigationView.getMenu().getItem(0).setCheckable(false);
```

Comment:
Is it possible to put the line 
`bottomNavigationView.getMenu().getItem(0).setCheckable(false);`
in the method 
`BottomNavigationListenerInstaller.install(bottomNavigationView, getActivity(), getContext());`

instead of in the calling fragment and avoid repeat the line in all fragments